packages/frontend/.eslintrc.json

```
@@ -0,0 +1,91 @@
+          "/@gen": "./src-generated"
+        },
+        "extensions": [".ts"],
+        "packages": ["packages/*"]
+      }
+    }
+  },
+  "ignorePatterns": [
+    "node_modules/**",
+    "**/dist/**"
+  ],
+  "plugins": ["@typescript-eslint", "sonarjs", "etc", "redundant-undefined", "no-null"],
+  "rules": {
+    "eqeqeq": "error",
+    "@typescript-eslint/no-unused-vars": ["error", { "argsIgnorePattern": "^_" }],
+    "@typescript-eslint/no-var-requires": "off",
+    "@typescript-eslint/consistent-type-imports": "error",
+    "@typescript-eslint/no-explicit-any": "error",
+    "prefer-promise-reject-errors": "error",
+    "@typescript-eslint/await-thenable": "error",
+    "@typescript-eslint/no-floating-promises": "error",
+    "@typescript-eslint/no-misused-promises": "error",
+    "@typescript-eslint/prefer-optional-chain": "error",
+    "no-null/no-null": "error",
+
+  /**
+   * Having a semicolon helps the optimizer interpret your code correctly.
+   * This avoids rare errors in optimized code.
+   * @see https://twitter.com/alex_kozack/status/1364210394328408066
+   */
+    "semi": [
+      "error",
+      "always"
+    ],
+    /**
+     * This will make the history of changes in the hit a little cleaner
+     */
+    "comma-dangle": [
+      "warn",
+      "always-multiline"
+    ],
+    /**
+     * Just for beauty
+     */
+    "quotes": ["error", "single", { "allowTemplateLiterals": true }],
+    "import/no-duplicates": "error",
+    "import/no-unresolved": "off",
+    "import/default": "off",
+    "import/no-named-as-default-member": "off",
+    "import/no-named-as-default": "off",
+    "sonarjs/cognitive-complexity": "off",
+    "sonarjs/no-duplicate-string": "off",
+    "sonarjs/no-empty-collection": "off",
+    "sonarjs/no-small-switch": "off",
+    "etc/no-commented-out-code": "error",
+    "etc/no-deprecated": "off",
+    "redundant-undefined/redundant-undefined": "error",
+    "import/no-extraneous-dependencies": "error"
+  }
+}
```

should raise a flag somewhere as it is missing an EOF newline